### ISSUE TITLE
add post to Community Announcements for 2022 PDS Customer Satisfaction Survey

### ISSUE THREAD

**Issue by c-suh** [NASA-PDS/portal-tasks#12]
requested by Laura Givens. Details sent me via email. CC Vivian (and Jordan?) on response with question of who exactly at PO to send the draft for review

**Comment by c-suh** [comment]
Emailed Laura with the page and clarifying question; also CC'ed Vivian

**Comment by jordanpadams** [comment]
@c-suh btw, there is also a "Task" template when you create a new issue for one-off tasks we have for each repo if you wanted to use them in the future. not required, but helps make sure I'm assigned and gets all the labels assigned and such.

https://github.com/NASA-PDS/portal-tasks/issues/new/choose 
<img width="1310" alt="Screen Shot 2022-02-22 at 1 36 48 PM" src="https://user-images.githubusercontent.com/33492486/155223733-e1f6e8b7-044c-4c95-96e8-097b6fe9c375.png">

**Comment by c-suh** [comment]
ah, I missed that. OK; got it! And if using a blank issue, assign you so that you can prioritize and assign labels and so on. :thumbsup:

**Comment by c-suh** [comment]
[Community Announcements](https://pds.nasa.gov/home/community-announcement.shtml) has been updated. Front page is TBD.

**Comment by jordanpadams** [comment]
revised banner announcement from @edwardbarraza 
![image](https://user-images.githubusercontent.com/33492486/155644243-5d134909-0cd6-4c24-82a5-cb9450e3754d.png)

**Comment by edwardbarraza** [comment]
https://www.figma.com/file/Sr1NxgRuxhTfHNncuCMV7y/Engineering-Node?node-id=2154%3A2693

**Comment by jordanpadams** [comment]
Per comments from @edwardbarraza above, the design team looked into ways to button-up our existing web site user experience in the interim, as they work towards a new design. Banners are a common web design module that should be consistent across the website.

Per email from Ed:
> Back before HDS was on our radar, we had discussed having alert banners (global, sectional, inline) on the PDS homepage. There obviously would have to be some categorization about what content gets displayed on each banner.
> 
> This to me feels like a sectional banner, in that it is not necessarily a global event (PDS downtime) but rather specific to a piece of content. 
> 
> The banner itself should be prominent, contain a short message related to the users action, and the action. We can show them more information once they’ve clicked the link. Without too much effort required, here is a cleaned up version of the banner taking these points in the previous paragraph in to account.

**Comment by c-suh** [comment]
Have let Tim and Laura know to review the front page and have asked if it should be made operational.

**Comment by edwardbarraza** [comment]
Looks good to me! Thanks @c-suh

**Comment by c-suh** [comment]
Operational at pds.nasa.gov. To be updated again once survey link is ready.

**Comment by jordanpadams** [comment]
closing since the initial announcement has been posted. adding link and updating text can be copy-pasted from this ticket, but that is a trivial task and I don't think this needs to remain open.
